bot/src/index.js

```diff
@@ -1,57 +1,48 @@
-const Feed = require("feed-to-json-promise");
 const { Client } = require("discord.js");
 
-const client = new Client();
-const feed = new Feed();
+const commands = require("./commands/index");
+const { getFeed } = require("./utilities/getFeed");
 
-const serverID = "458034851296313375";
+const client = new Client();
+const feedList = ["https://xkcd.com/rss.xml"];
+const serverId = "458034851296313375";
 
 client.on("ready", async () => {
   console.log(`Logged in as ${client.user.tag}!`);
-  const rssFeed = "https://xkcd.com/rss.xml";
-  const channelId = await addFeedChannel(rssFeed, serverID);
-  try {
-    const items = await getFeedItems(rssFeed);
-    sendFeedItems(items, channelId);
-  } catch (error) {
-    console.error(error);
+  for (const rssFeed of feedList) {
+    try {
+      await commands.add([rssFeed], serverId, client);
+    } catch (error) {
+      console.error(error);
+    }
   }
 });
 
 client.on("message", msg => {
   if (msg.content === "ping") {
     msg.reply("pong");
   }
+  if (!msg.mentions.users.size) {
+    return;
+  } else if (msg.mentions.users.first().username === "RnD") {
+    const args = msg.content.split(/\s+/);
+    args.shift();
+    runCommand(args, msg);
```

Comment:
You should `return` the Promise from `runCommand()` in case `discord.js` wants to wait for it.

In general you should usually avoid dangling Promises, or you could end up with confusing concurrency bugs.